internal/testutil/tempdir.go

```diff
@@ -1,9 +1,6 @@
-// Copyright 2020-2022 the Pinniped contributors. All Rights Reserved.
+// Copyright 2020-2023 the Pinniped contributors. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-//go:build !go1.14
```

Comment:
These tags actually seem to contradict themselves, its not immediately obvious to me what they were doing.

Reply:
I agree, I find the `!go1.XX` tag mechanism confusing as well.